From: Kefu Chai <kchai@redhat.com>
Date: Wed, 31 Aug 2016 13:11:24 +0800
Subject: mon/AuthMonitor: make AuthMonitor::IncType public

so ceph-objectstore-tool is able to use it when rebuilding monitor
db.

Fixes: http://tracker.ceph.com/issues/17179
Signed-off-by: Kefu Chai <kchai@redhat.com>
(cherry picked from commit 19ef4f16b3aba04119ac647cd6261c74a57ce829)

Resolves: rhbz#1374224
---
 src/mon/AuthMonitor.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mon/AuthMonitor.h b/src/mon/AuthMonitor.h
index 4420290..e4a6bb6 100644
--- a/src/mon/AuthMonitor.h
+++ b/src/mon/AuthMonitor.h
@@ -35,11 +35,11 @@ class KeyRing;
 #define MIN_GLOBAL_ID 0x1000
 
 class AuthMonitor : public PaxosService {
+public:
   enum IncType {
     GLOBAL_ID,
     AUTH_DATA,
   };
-public:
   struct Incremental {
     IncType inc_type;
     uint64_t max_global_id;
